velox/exec/Unnest.cpp

```diff
@@ -150,17 +198,29 @@ void Unnest::generateRepeatedColumns(
     vector_size_t start,
     vector_size_t size,
     vector_size_t numElements,
-    std::vector<VectorPtr>& outputs) {
+    std::vector<VectorPtr>& outputs,
+    vector_size_t firstRowEndSize,
+    vector_size_t endRowEndSize) {
   // Create "indices" buffer to repeat rows as many times as there are elements
   // in the array (or map) in unnestDecoded.
   auto repeatedIndices = allocateIndices(numElements, pool());
   auto* rawRepeatedIndices = repeatedIndices->asMutable<vector_size_t>();
   vector_size_t index = 0;
-  for (auto row = start; row < start + size; ++row) {
+  if (size > 0) {
```

Comment:
Is it possible for size to be 0? When would this happen? Would be nice to clarify in a comment.